io.github.aganzha.Stage.json

```diff
@@ -0,0 +1,45 @@
+{
+   "app-id": "io.github.aganzha.Stage",
+   "runtime": "org.gnome.Platform",
+   "runtime-version": "47",
+   "sdk": "org.gnome.Sdk",
+   "sdk-extensions": [
+       "org.freedesktop.Sdk.Extension.rust-stable"
+   ],
+   "command": "stage",
+   "finish-args": [
+      "--filesystem=home",
+      "--share=ipc",
+      "--share=network",
+      "--socket=fallback-x11",
+      "--socket=wayland",
+      "--socket=ssh-auth",
+      "--socket=gpg-agent",
+      "--device=dri"
+   ],
+   "build-options": {
+      "append-path": "/usr/lib/sdk/rust-stable/bin"
+   },
+   "modules": [{
+      "name": "stage",
+      "buildsystem": "simple",
+      "build-options": {
+         "env": {
+            "CARGO_HOME": "/run/build/stage/cargo"
+         }
+      },
+      "build-commands": [
+         "cargo fetch --manifest-path Cargo.toml --offline --verbose",
+         "cargo build --release --verbose --offline",
+         "source ./install.sh"
```

Comment:
a shell script is not a buildsystem replacement.

Reply:
- fixed hardcoded path https://github.com/aganzha/stage/commit/ad73dd3b92cce54497d7ad9d0b9b4dd6add084b3
- fixed missing icon https://github.com/aganzha/stage/commit/8e29799979c00fa3c516b18716e36cf5a71b1747
- switched to meson build and bumped version (changes in this PR)